src/Features/Core/Portable/AddImport/CodeActions/SymbolReference.SymbolReferenceCodeAction.cs

```diff
@@ -23,36 +20,113 @@ private partial class SymbolReference
             /// </summary>
             private class SymbolReferenceCodeAction : CodeAction
             {
-                private readonly AsyncLazy<CodeActionOperation> _getOperation;
```

Comment:
No lazy callbacks in these code actions.  The oop server will compute everything necessary and will pass that back to VS to just display.